content/_data/books/Mastering_Jenkins.yml

```
@@ -0,0 +1,19 @@
+title: "Mastering Jenkins"
+url: https://www.amazon.com/Mastering-Jenkins-Jonathan-McAllister/dp/1784390895/
+author: "Jonathan McAllister"
+publication_year: 2015
+publisher: "Packt Publishing"
+type: Paid
+image: books/Mastering-Jenkins.jpg
+description: |-
+  Configure and extend Jenkins to architect, build, and automate efficient software delivery pipelines 
+  . Configure and horizontally scale a Jenkins installation to support a development organization of any size 
+  . Implement Continuous Integration, Continuous Delivery, and Continuous Deployment solutions in Jenkins 
+  . A step-by-step guide to help you get the most out of the powerful automation orchestration platform that is Jenkins 
+  . Create and manage various types of build jobs, and implement automation tasks to support a software project of any kind 
+  . Get to grips with the automated testing architecture, and scalable automated testing techniques 
+  . Facilitate the delivery of software across the SDLC by creating scalable automated deployment solutions 
+  . Manage scalable automation pipelines in Jenkins using the latest build, test, and deployment strategies 
+  . Implement a scalable master / slave build automation platform, which can support Windows, Mac OSX, and Linux software solutions 
+  . Cover troubleshooting and advanced configuration techniques for Jenkins slave nodes. 
```

```suggestion
Learn how to configure and extend Jenkins to architect, build, and automate efficient software delivery pipelines.
You'll review how to configure and horizontally scale a Jenkins installation, to support a development organization of any size.
This book also describes how to implement Continuous Integration, Continuous Delivery, and Continuous Deployment solutions in Jenkins.
Discover how automated testing architecture and scalable automated testing techniques can be used to take your Jenkins builds to the next level.
You'll read about how to manage scalable automation pipelines in Jenkins using the latest build, test, and deployment strategies.
```
updated formatting/syntax/length as in other suggestions.

I also deleted any mentions/usage of the words "master" and "slave" as they are no longer accepted terminology and have been replaced by "controller" and "agent".